src/hooks/useSyncChainQuery.ts

```diff
@@ -31,8 +32,19 @@ export default function useSyncChainQuery() {
   const [searchParams, setSearchParams] = useSearchParams()
 
   useEffect(() => {
-    if (isActive && urlChainId && chainId !== urlChainId) {
+    // Change a user's chain on pageload of the connected chainId does not match the query param chain
```

Comment:
If this is only meant to run on pageload, should we set it as a mount-only effect (by setting the deps to an empty array)?

Reply:
We added deps because connected wallets load on a delay and would overwrite this setting with their chain and user wouldn't be prompted to switch